lib/cli.rb

```diff
@@ -477,9 +489,11 @@ def self.parse_scopes(scope_string)
 
       filter = FilterOptionParser.parse("inspect", options, global_options)
 
-      if options["verbose"] && !filter.empty?
+      if options[:verbose] && !filter.empty?
```

Comment:
You're using a symbol here but a string a few lines above. Do both work? But even if both work I think it would be cleaner to use one or the other consistently.